libcaf_core/CMakeLists.txt

```diff
@@ -129,7 +129,8 @@ if (NOT CAF_BUILD_STATIC_ONLY)
                         OUTPUT_NAME caf_core)
   if (CYGWIN)
     install(TARGETS libcaf_core_shared RUNTIME DESTINATION bin)
-  elseif (NOT WIN32)
+  else()
+# elseif (NOT WIN32)
```

Comment:
Please remove lines instead of commenting them out. There's no reason to keep them around.